html/admin/project/crew/vacantCrew.twig

```diff
@@ -151,28 +144,50 @@
                             </div>
                             <div class="input-group" style="margin-bottom: 5px;">
                                 <div class="input-group-prepend">
-                                    <span class="input-group-text">Collect Phone Numbers</span>
+                                    <span class="input-group-text" data-toggle="tooltip" data-placement="bottom" title="Allow people to assign themselves to this role, and automatically close the role once it has been filled">
```

Comment:
Do the tooltips work on mobile?

Reply:
Yeah seem to for me.  

 ![image](https://user-images.githubusercontent.com/54247748/157710062-c9325621-a05a-4b22-9ced-979d5949ccfb.png)

Reply:
Omg amazing - looks sexy too